### ISSUE TITLE
Narrator remains silent after activating Custom XamlUICommand app bar button in XamlUICommand.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/WinUI-Gallery#169]
**Describe the bug**

**Pre-requisite:** 
Turn on Narrator (Keyboard Shortcut: Ctrl+Win+Enter).
Set Narrator to item mode.

**Steps to reproduce the bug**

1.Launch the XAML Controls Gallery Application.
2. Activate XamlUICommand from All Controls Page.
3. Navigate to "Custom XamlUICommand" app bar button and activate it and observe narrator reading.

**Expected behavior**

After activating the "Custom XamlUICommand" narrator should give the information.

**Actual behavior**

Narrator remains silent without reading anything.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/63801487-40a24680-c8c5-11e9-8bd0-c0d1a40a868c.png)

**Version Info**

Gallery version (found on Settings page): 

Version Info: 1.2.6.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/20822597)

**Comment by chingucoding** [comment]
I agree that this is not a behavior that we should leave in the app, however what should the narrator read out? And more importantly, how should the narrator read it? Should the focus change to an item with text?

Also: The Button examples behave the same, meaning activating them does not provide a feedback via the narrator. This should be addressed too.
